README.md

```diff
@@ -11,7 +11,7 @@
 # Dockstore
 
 Dockstore provides a place for users to share tools encapsulated in Docker and described with the Common 
-Workflow Language (CWL), WDL (Workflow Description Language), or Nextflow. This enables scientists to share analytical 
+Workflow Language (CWL), WDL (Workflow Description Language), Galaxy, or Nextflow. This enables scientists to share analytical 
```

Comment:
Add Snakemake, also?

Reply:
I dunno, technically the CLI doesn't do Snakemake (or Galaxy), so leaning toward removing. I mainly made this change to kick off the build since it looked like the workspace was broken,